portifolio.html

```diff
@@ -61,5 +61,83 @@ <h2>Software Developer</h2>
 </div>
+
+<div class="arrow">
+    <img src="image/arrow.svg" alt="down-arrow" />
+    <h3>Work</h3>
+</div>
+<div>
+    <h2 class="work">MY RECENT WORK</h2>
+    <img src="image/Vector-4.png" alt="bar" class="img">
+</div>
+<section class="work-section">
+    <div class="placeholder">
+        <div>
+            <img src="image/placeholder.png" alt="blank-placeholder">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        </div>
+        <div>
+            <ul class="items">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+        </div>
+    </div>
+    
+    <div class="background-image1">
+        <div>
+            <img src="image/placeholder.png" alt="blank-placeholder">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        </div>
+        <div>
+            <ul class="items">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+        </div>
+    </div>
+
+    <div class="background-image2">
+        <div>
+            <img src="image/placeholder.png" alt="blank-placeholder">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        </div>
+        <div>
+            <ul class="items">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+        </div>
+    </div>
+
+    <div class="background-image3">
+        <div>
+            <img src="image/placeholder.png" alt="blank-placeholder">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        </div>
+        <div>
+            <ul class="items">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+            </ul>
```

Comment:
[OPTIONAL]
- This may be marked as optional but you have to fix this. Currently, your technologies are not displaying correctly as seen below 
![image](https://user-images.githubusercontent.com/45460192/165790688-87384a61-c07b-4711-8ad1-bd279093aadb.png). Please fix that according to the figma 👍